workflows_core/engine/stable_engine.py

```diff
@@ -32,11 +33,18 @@ def apply(self) -> float:
             try:
                 new_batch = self.operator(chunk)
             except Exception as e:
+                chunk_error_log = {
+                    "exception": str(e),
+                    "traceback": traceback.format_exc(),
+                    "chunk": chunk,
```

Comment:
Don't store the chunk or this will break metadata - it can't store too much info. Only store the ID of the chunk.